brave/brave-core
# Always enable cosmetic filtering for playlist's bg webcontents

fix https://github.com/brave/brave-browser/issues/25366

By enable cosmetic filtering always, we can get media contents w/o ads regardless of shields setting state.


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
`CosmeticFilteringPlaylistFlagEnabledTest.AllowCosmeticFiltering`

1. Launch brave and enable playlist flags from brave://flags
2. Re-launch and load chrome-untrusted://playlist
3. Disable ads-block for youtube or global shields settings
4. Copy any youtube url to playlist webui page
5. Check media contents is always downloaded w/o ads
